java/core/src/main/java/com/whylogs/core/metrics/IntegralMetric.java

```diff
@@ -62,7 +63,14 @@ public OperationResult columnarUpdate(PreprocessedColumn data) {
     return OperationResult.ok(successes);
   }
 
-  // @Override // TODO: move this to a factory so it can be accessed for building
+  @Override
```

Comment:
On line 30 I think the components should be copied rather than keeping references to them.